src/js/script.js

```diff
@@ -32,8 +32,21 @@
       },
     },
 
+    cartProduct: {
+      amountWidget: '.widget-amount',
+      price: '.cart__product-price',
+      edit: '.cart__product-edit',
+      remove: '.cart__product-remove',
+    },
+
     cart: {
-      toggleTrigger: '.cart__summary', 
+      productList: '.cart__order-summary',
+      toggleTrigger: '.summary',
```

Comment:
Powinno być `'.cart__summary`. Klasa `summary` nie istnieje w elemencie cart dlatego, jak próbujesz złapać element z tą klasa, to zwraca `undefined`
```suggestion
      toggleTrigger: '.cart__summary',
```